islandora_pathauto.module

```diff
@@ -187,6 +190,13 @@ function islandora_pathauto_pathauto_bulkupdate(&$context) {
       $filter = "sameTerm(?model, <info:fedora/$filter>)";
     }
     $sandbox['filters']['model'] = format_string('FILTER(!models)', array('!models' => implode(' || ', $model_filters)));
+    if (variable_get('islandora_namespace_restriction_enforced', FALSE)) {
```

Comment:
@DiegoPino  - it seems you have to change this review from "requested changes"? (still getting used to this github review business)